scripts/eurostat/life_expectancy_by_age_gender/preprocess.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import re
+import sys
+path = 'demo_r_mlifexp.tsv'
+
+def group_to_wide_form(data, col):
+    """reorgnize the data of each statvars into independent columns """
+    data_grouped = data.groupby([col])
+    subsets = []
+    for _, subset in data_grouped:
+        pivot = subset[col].iloc[0]
+        subset = subset.rename(columns = {'geo': pivot+'_geo',
+            'year': pivot + '_year', 'life_expectancy': pivot})
+        subset = subset.drop(columns = [col]).reset_index(drop=True)
+        subsets.append(subset)
+    data = pd.concat(subsets, axis=1, join = 'outer')
+    return data
+
+def nuts_to_iso(data):
+    """convert nuts code to ISO Alpha3 code"""
+    # no need in the future, can be resolved through ISO Alpha 2
```

Comment:
https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar

Reply:
removed this line.